src/pykiso/lib/connectors/cc_socket_can/cc_socket_can.py

```diff
@@ -89,29 +89,40 @@ def __init__(
         self.logging_activated = logging_activated
         self.bus = None
         self.logger = None
-        self.log_path = log_path
-        self.log_name = log_name
+        self.trace_path = trace_path
+        self.log_name = log_name.strip(".trc") if log_name else None
         # Set a timeout to send the signal to the GIL to change thread.
         # In case of a multi-threading system, all tasks will be called one after the other.
         self.timeout = 1e-6
 
         if self.logging_activated:
-            # Just avoid the case the given trace directory is None
-            self.log_path = "" if self.log_path is None else self.log_path
+            # if no trace path is given, take the root path (where pykiso is launched)
+            if self.trace_path is None or self.trace_path == "":
+                self.trace_path = Path().resolve()
+
             # if the given log path is not absolute add root path
             # (where pykiso is launched) otherwise take it as it is
-            dir_path = (
-                (Path() / self.log_path).resolve() if not Path(self.log_path).is_absolute() else Path(self.log_path)
-            )
-            # if no specific logging file name is given take the default one
-            self.log_name = (
-                time.strftime(f"%Y-%m-%d_%H-%M-%S_{self.log_name}.trc")
-                if self.log_name is not None
-                else time.strftime("%Y-%m-%d_%H-%M-%S_CanLog.trc")
+            self.trace_path = (
+                (Path().resolve() / self.trace_path).resolve()
```

Comment:
i think the outer resolve has no effect

Reply:
could you just do Path(self.trace_path).resolve()

Reply:
yupp